packages/clerk-js/src/core/clerk.ts

```diff
@@ -197,6 +197,26 @@ export default class Clerk implements ClerkInterface {
     return this.#instanceType;
   }
 
+  get canUseCaptcha(): boolean | undefined {
```

Comment:
Let's rename this to 
```suggestion
  get experimental_canUseCaptcha(): boolean | undefined {
```